components/org.wso2.carbon.identity.discovery/src/main/java/org/wso2/carbon/identity/discovery/internal/OIDCDiscoveryServiceComponent.java

```diff
@@ -55,6 +58,13 @@ protected void activate(ComponentContext context) {
         }
     }
 
+    @Reference(
+            name = "claim.manager.listener.service",
+            service = ClaimMetadataManagementService.class,
+            cardinality = ReferenceCardinality.MULTIPLE,
```

Comment:
Seems this should be mandatory. It's converted correctly, but the previous one was wrong.